### DISCUSSION TITLE
I do not have the right to vote on the snapshot, stake in uniswap

### DISCUSSION THREAD

**Discussion by malcevg17** [snapshot-labs/snapshot#3663] [Q&A]
I do not have the right to vote on the snapshot, stake in uniswap, please help
0xC5b1ddB34cB01E3CA9Ae6dd7cD90DFf5606cb508
https://prnt.sc/9CoIqdhTqN7M
https://prnt.sc/k6Isp0j1Nt5X

**Comment by sadlymad** [comment]
me too

**Comment by FedorSeleninov** [comment]
the same thing

**Comment by 1nikkas** [comment]
I couldn't figure it out either
